src/test/java/core/WinnersTest.java

```diff
@@ -0,0 +1,54 @@
+package core;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class WinnersTest {
+
+    @Test
+    @DisplayName("position 이 가장 큰 차들이 우승자가 된다 1")
+    void decideWinners1() {
+        //given
+        Car car1 = TestObjectGenerator.generateCar("a", 4);
+        Car car2 = TestObjectGenerator.generateCar("b", 3);
+        Car car3 = TestObjectGenerator.generateCar("c", 2);
+        Cars cars = Cars.fromCars(Arrays.asList(car1, car2, car3));
+
+        List<CarName> winnerCarNames = new ArrayList<>();
+        winnerCarNames.add(CarName.create("a"));
+
+        //when
+        Winners winners = Winners.decideWinners(cars);
+
+        //then
+        assertThat(winners.getCars().size()).isEqualTo(1);
+        winners.getCars().forEach(car -> assertThat(winnerCarNames).contains(car.getCarName()));
```

Comment:
```suggestion
        CarName expected = CarName.create("a");

        //when
        Winners winners = Winners.of(cars);

        //then
        assertThat(winners.size()).isEqualTo(1);
        assertThat(winners.names()).contains(expected);
```
- 값을 꺼내서 비교하시는 경우가 많은데요. 디미터 원칙을 위반할 경우에 어떤 문제가 있을지도 한번 고민해보셔요.